pr-data.csv

```diff
@@ -413,10 +413,10 @@ https://github.com/apache/avro,bfbd2d115aec576545b0673e876a652806b41986,lang/jav
 https://github.com/apache/avro,bfbd2d115aec576545b0673e876a652806b41986,lang/java/avro,org.apache.avro.reflect.TestReflect.testMultipleFieldAliases,ID,,,
 https://github.com/apache/avro,bfbd2d115aec576545b0673e876a652806b41986,lang/java/avro,org.apache.avro.reflect.TestReflect.testRecord,ID,Accepted,https://github.com/apache/avro/pull/667,
 https://github.com/apache/bookkeeper,2aa59c74cdd1de13c33a593f6dccc888823c4a83,bookkeeper-server,org.apache.bookkeeper.replication.AuditorReplicasCheckTest.testReplicasCheckForBookieHandleNotAvailable,ID,,,
```

Comment:
I'm wondering did your PR `https://github.com/apache/bookkeeper/pull/3551` fix this test? since you also modified the code in this test.

Reply:
Yes, it did - apologies for missing it.
I have updated the PR.